### ISSUE TITLE
Dark mode - CSAT rating options do not have visible text in dark mode

### ISSUE THREAD

**Issue by chandylan** [microsoft/BotFramework-WebChat#5404]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Dark mode users will not be able to distinguish rating options

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

Yes, however, this also repro when customizations are not in use.

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Contrast ratio

### What is the public URL for the website?

copilot studio test chat

### How to reproduce the issue?

1. Login to Copilot Studio and open an Agent
2. Click on settings->General setting->toggle on the Dark mode on button
3. In the test chat, trigger a topic which has end of conversation (e.g. send "goodbye")
4. Click the "yes" quick reply until you reach the CSAT message ("Great! Please rate your experience" with stars labeled 1-5)

![Image](https://github.com/user-attachments/assets/5ee821a5-008a-4298-8a97-5d6887ec1dce)

### What do you expect?

CSAT rating numbers are visibly clear (e.g. white or light gray)

here is how the numbers look in light mode:
![Image](https://github.com/user-attachments/assets/b6867728-457a-4c18-ac3d-c05473c8b604)

### What actually happened?

CSAT rating numbers are not clear:

![Image](https://github.com/user-attachments/assets/7dadc2e4-e511-48dd-be52-85860c61c6fe)

This is likely because the CSAT options are static images with black text

![Image](https://github.com/user-attachments/assets/b62fb52e-5366-4ef2-a96f-09da2e8ee0ff)

### Do you have any screenshots or recordings to repro the issue?

see above

### Did you find any DOM elements that might have caused the issue?

see above

### Adaptive Card JSON

```json

```

**Comment by OEvgeny** [comment]
The adaptive card is not a part of Web Chat.
